hare/haretypes.go

```diff
@@ -71,12 +71,8 @@ func (b32 Bytes32) Bytes() []byte {
 }
 
 func (b32 Bytes32) String() string {
-	// TODO: convert the whole 32 bytes to string number
-	s := uint64(0)
-	for x := range b32.Bytes() {
-		s += uint64(x)
-	}
-	return string(s)
+	// TODO: should improve
+	return string(b32.Id())
```

Comment:
I'd rather instanceID to return a number so we could compare it to layer numbers printed from other modules.